src/client/config/menuItems.ts

```diff
@@ -106,6 +106,12 @@ const myChaptersMenuItems: ChapterMenuItem[] = [
     to: AppRoutes.SIA,
     profileTypes: ['private', 'private-commercial', 'commercial'],
   },
+  {
+    title: ChapterTitles.TOERISME,
+    id: Chapters.TOERISME,
+    to: AppRoutes.TOERISME,
+    profileTypes: ['private', 'private-commercial', 'commercial'],
```

Comment:
Kun je even bij Marian checken of dit ook voor zakelijke klanten (commercial) en zzp'ers (private-commercial) geldt? Ik denk dat we (nog) geen bevragingen doen op basis van KVK nummer maar alleen BSN